cylc/flow/option_parsers.py

```diff
@@ -149,15 +149,12 @@ def add_std_options(self):
             self.add_std_option(
                 '--color', '--colour', metavar='WHEN', action='store',
                 default='auto', choices=['never', 'auto', 'always'],
-                help='Determine when to use color in terminal output.')
-
-        if self.prep:
-            self.add_std_option(
-                "--suite-owner",
-                help="Specify suite owner",
-                metavar="OWNER", action="store", default=None,
-                dest="suite_owner")
-
+                help=(
+                    "Determine when to use color in terminal output. "
+                    "Options available are 'never', 'auto' and 'always'. "
+                    "'never' will also remove bold typeface."
```

Comment:
Bold text isn't specific to `--color=never` (awkwardly it doesn't necessarily render as bold the terminal either):

```suggestion
                    "Determine when to use color/bold text in terminal output. "
                    "Options available are 'never', 'auto' and 'always'."
```